src/extension/linker.js

```diff
@@ -0,0 +1,66 @@
+import path from 'path';
+import bluebird from 'bluebird';
```

Comment:
I like to seperate bluebird from node's integrated Promise implementation.
This allows me to depend on bluebird only for functionality which is missing from node itself.